README.md

```diff
@@ -96,6 +96,15 @@ Please create issues in this repo for any problems or questions you find. Before
 
 We're still in the process of getting everying running 100%, but please refer to the [Serverless contributing guidlines](https://github.com/serverless/serverless/blob/master/CONTRIBUTING.md) for information on how to contribute and code of conduct.
 
+#### Local dev
+
+1. Clone this repository to your local machine
```

Comment:
Nit: Update to either numbered steps that are different numbers (not just 1) or bullet points

Reply:
Done

Reply:
In markdown, you can just name everything as `1.` and it will auto number when render. 

For example, this will be auto-numbered in Preview tab.

1. one
1. two
1. three